spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :user do
+    email Faker::Internet.email
```

Comment:
you can drop faker and always generate similar email but use `sequence` from factory bot in it
sth like "email_#{n}@example.com"

Reply:
same as above